exam-01/components/checkAll.js

```diff
@@ -0,0 +1,53 @@
+class CheckAll {
+    #container;
+    #data;
+    #notifyAllCheck;
+
+    constructor(container, data, notifyAllCheck) {
+        this.#container = container;
+        this.#data = data;
+        this.#notifyAllCheck = notifyAllCheck;
+
+        this.#container.addEventListener('click', this.#onCheckAllClick.bind(this));
+    }
+
+    render() {
+        let itemsLeft = this.#data.querySelectorAll('li');
```

Comment:
Be very careful with naming the variables. When hearing the word `data`, I would think it's real `data` object or array which contains NO UI element. However, in this case, your `#data` actually is a DOM element. This is not a good name